config/codemod-let-name.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Restores var names transformed by babel's let block scoping
+ */
+export default (file, api) => {
+	let j = api.jscodeshift;
+	let code = j(file.source);
+
+	code.findVariableDeclarators().filter(d => /^_i/.test(d.value.id.name)).renameTo('i');
```

Comment:
Yes it would be possible if someone added a var with that prefix. No scope analysis. 

I personally wouldn't add this mod at all. I just wanted to make the output identical.